robottelo/cli/settings.py

```diff
@@ -28,3 +29,14 @@ def set(cls, options=None):
         cls.command_sub = 'set'
 
         return cls.execute(cls._construct_command(options))
+
+    @classmethod
+    def info(cls, options=None, output_format=None, return_raw_response=None):
+        """Show setting info"""
+        cls.command_sub = 'info'
+
+        return cls.execute(
+            cls._construct_command(options),
+            output_format=output_format,
+            return_raw_response=return_raw_response,
+        )
```

Comment:
We already have info classmethod defined in the Base class, which Settings class is already inheriting, so defining this again could be skipped